fonts/fonts.css

```diff
@@ -0,0 +1,26 @@
+@font-face {
+	font-family: 'montserrat';
+	src: url('/fonts/bold/montserrat-bold-webfont.woff2') format('woff2'),
+		url('/fonts/bold/montserrat-bold-webfont.woff') format('woff');
+	font-display: swap;
+	font-weight: bold;
+	font-style: normal;
+}
+
+@font-face {
+	font-family: 'montserrat';
+	src: url('/fonts/regular/montserrat-regular-webfont.woff2') format('woff2'),
+		url('/fonts/regular/montserrat-regular-webfont.woff') format('woff');
+	font-display: swap;
+	font-weight: normal;
```

Comment:
Codacy found an issue: [Expected indentation of 2 spaces (indentation)](https://app.codacy.com/gh/WebHeroSchool/GurevAleksey_itogovaya/pullRequest?prid=6087879)